<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>861</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>This Weeks Games</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>881</width>
      <height>591</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>879</width>
       <height>589</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>881</width>
        <height>591</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QScrollBar" name="verticalScrollBar">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>20</width>
            <height>591</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNFL">
    <property name="title">
     <string>NFL</string>
    </property>
    <addaction name="actionSChedule"/>
    <addaction name="separator"/>
    <addaction name="actionHistory"/>
    <addaction name="separator"/>
    <addaction name="actionData"/>
   </widget>
   <widget class="QMenu" name="menuMLB">
    <property name="title">
     <string>MLB</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNCAAF">
    <property name="title">
     <string>NCAAF</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNBA">
    <property name="title">
     <string>NBA</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPremiere_League">
    <property name="title">
     <string>Premiere League</string>
    </property>
   </widget>
   <addaction name="menuNFL"/>
   <addaction name="menuMLB"/>
   <addaction name="menuNCAAF"/>
   <addaction name="menuNBA"/>
   <addaction name="menuPremiere_League"/>
  </widget>
  <action name="actionNFL">
   <property name="text">
    <string>NFL</string>
   </property>
  </action>
  <action name="actionNBA">
   <property name="text">
    <string>NBA</string>
   </property>
  </action>
  <action name="actionNCAA_Football">
   <property name="text">
    <string>NCAA Football</string>
   </property>
  </action>
  <action name="actionPremiere_League">
   <property name="text">
    <string>Premiere League</string>
   </property>
  </action>
  <action name="actionMLB">
   <property name="text">
    <string>MLB</string>
   </property>
  </action>
  <action name="actionChampions_League">
   <property name="text">
    <string>Champions League</string>
   </property>
  </action>
  <action name="actionSChedule">
   <property name="text">
    <string>Schedule</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionData">
   <property name="text">
    <string>Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
